be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -163,16 +163,8 @@ Status HdfsTextScanner::do_init(RuntimeState* runtime_state, const HdfsScannerPa
     // In Hive, users can specify collection delimiter and mapkey delimiter as string type,
     // but in fact, only the first character of the delimiter will take effect.
     // So here, we only use the first character of collection_delim and mapkey_delim.
-    if (text_file_desc.collection_delim.empty() || text_file_desc.mapkey_delim.empty()) {
-        // During the StarRocks upgrade process, collection_delim and mapkey_delim may be empty,
-        // in order to prevent crash, we set _collection_delimiter
-        // and _mapkey_delimiter a default value here.
-        _collection_delimiter = '\002';
-        _mapkey_delimiter = '\003';
-    } else {
-        _collection_delimiter = text_file_desc.collection_delim.front();
-        _mapkey_delimiter = text_file_desc.mapkey_delim.front();
-    }
+    _collection_delimiter = text_file_desc.collection_delim.front();
```

Comment:
Does broker load respect this protocol?

Reply:
broker load is using `csv_scanner` not `hdfs_scanner_text`

Reply:
OK. We can use default value here for `field_delim` and `line_delim` too instead of return error. What do you think.